lib/Wikia/tests/Service/User/Auth/MediaWikiAuthServiceTest.php

```diff
@@ -13,7 +13,7 @@ public function setUp() {
 		parent::setUp();
 
 		$this->heliosClientMock = $this->getMock( HeliosClient::class,
-			[ 'login', 'forceLogout', 'invalidateToken', 'register', 'info', 'generateToken' ] );
+			[ 'login', 'forceLogout', 'invalidateToken', 'register', 'info', 'generateToken' , 'setPassword', 'validatePassword' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "login" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)